third_party/cpgfunctionEP-0.3.2/include/cpgfunction/utilities.h

```diff
@@ -0,0 +1,41 @@
+//
```

Comment:
These are functions associated with creating a time vector for input to the g-function calculation.